sormas-backend/src/test/resources/META-INF/persistence.xml

```diff
@@ -105,6 +105,8 @@
 			<property name="hibernate.show_sql" value="false" />
 			<property name="hibernate.format_sql" value="false" />
 
+			<property name="hibernate.enable_lazy_load_no_trans" value="true" />
```

Comment:
This solution is in place already for the UI tests. Possible other solutions would be:
- replace beantest ( as it creates the transactions kind of chaotically - tried debugging it at some point but ended up with this solution)
- using only localbeans for the AbstractBeanTest get...Facade methods might solve the issue with when beantest creates the transaction - only a maybe and it is a big effort
If you do have other suggestions please let us know. 
Note that it is for the tests and not for the production code, there we definitely shouldn't do this, but no need either since transactions are correctly created there.